tests/test_configure_feedstock.py

```diff
@@ -815,3 +815,33 @@ def test_cos7_env_render(py_recipe, jinja_env):
         else:
             if "DEFAULT_LINUX_VERSION" in os.environ:
                 del os.environ["DEFAULT_LINUX_VERSION"]
+
+
+def test_cuda_enabled_render(cuda_enabled_recipe, jinja_env):
+    has_env = "CF_CUDA_ENABLED" in os.environ
```

Comment:
I had removed this line because I wasn't modifying `forge_config`, but it seems it's necessary now.
```suggestion
    forge_config = copy.deepcopy(cuda_enabled_recipe.config)
    has_env = "CF_CUDA_ENABLED" in os.environ
```